InOffice/View/HomeViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  HomeViewController.swift
+//  InOffice
+//
+//  Created by Luka Muše on 09.12.2021..
+//
+
+import UIKit
+
+
+
+class HomeViewController: UIViewController {
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureView()
+        tableView.reloadData()
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    lazy var tableView: UITableView = {
+        let table = UITableView()
+        table.delegate = self
+        table.dataSource = self
+        table.register(CustomCell.self, forCellReuseIdentifier: "\(CustomCell.self)")
+        table.register(HeaderSectionCell.self, forHeaderFooterViewReuseIdentifier: "\(HeaderSectionCell.self)")
+        table.rowHeight = 80.0
```

Comment:
Unnecessary, cell height is overriden in heightForHeaderInSection tableview method